index.html

```diff
@@ -14,6 +14,89 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+  <div class="body-container">
+    <header>
+      <nav>
+        <a href="#"> Home</a>
+        <a href="#"> About</a>
+        <a href="#"> Portfollo</a>
+        <a href="#contact"> Contact</a>
+      </nav>
+    </header>
+    <main>
+      <div class="container main-container">
+        <img src="images/vince.jpg" alt="my image">
+        <h1>Nmeregini Vincent - Web Dev</h1>
+        <p>Technology is my jam!</p>
+        <div>
+            <p>Occasionally I have time to work on side projects, if you're let me 
+                know by signing up on this wbsite let me know by signing up on this wbsite
+              </p>
+              <button><a href="#contact">Lets Team Up</a></button>
```

Comment:
I don't love the use of both a button and an anchor tag here. I think you should just stick to one of them here.
Later in the course you'll discover on how to trigger them after user interaction.